dvc/config_schema.py

```diff
@@ -179,10 +179,12 @@ class RelPath(str):
                 },
                 "hdfs": {"user": str, "kerb_ticket": str, **REMOTE_COMMON},
                 "webhdfs": {
-                    "hdfscli_config": str,
-                    "webhdfs_token": str,
-                    "user": str,
-                    "webhdfs_alias": str,
+                    "kerberos": Bool,
+                    "kerberos_principal": str,
+                    "proxy_to": str,
```

Comment:
Should it be `proxy_user` or `superuser` per https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/Superusers.html ? "proxy to" could refer to many things...

Reply:
Or `hadoop_proxy_user`

Reply:
Yes for this initial pull request I just copied the names of parameters from `fsspec` and left them as a first draft to have a discussion around.

I think `proxy_user` would be clearer, and in the hadoop docs I feel like I see proxy user more often than superuser.

If we do something like `hadoop_proxy_user` I feel like maybe we should prefix all the other options with `hadoop` or `webhdfs` as well to be consistent?

It seems like the other DVC protocol config options do not do this kind of prefixing except google drive, but I can see how it would make sense.

Reply:
Thanks @gudmundur-heimisson . It's an interesting question and I'm sure @efiop will know best what to do. I personally do like the prefixing idea